src/pages/index.tsx

```diff
@@ -30,122 +31,149 @@ import Layout from '~/components/Layout'
 
 const MainSection = () => (
   <Stack
-    px="10"
-    align="center"
+    alignItems="center"
+    justifyContent="center"
     direction={{ base: 'column-reverse', lg: 'row' }}
+    spacing={{ base: 10, lg: 1 }}
   >
-    <Stack
-      shouldWrapChildren
-      alignItems={{ base: 'center', lg: 'start' }}
-      spacing="8"
-      flexGrow={1}
-    >
+    <Stack alignItems={{ base: 'center', lg: 'start' }} spacing="24">
       <Heading
         as="h2"
         color="primaryText"
-        fontSize="4xl"
+        fontSize={{ base: '3xl', md: '5xl' }}
         fontWeight="black"
         textAlign={{ base: 'center', lg: 'left' }}
         letterSpacing="tight"
+        px={{ base: 12, lg: 0 }}
       >
-        {BRIEF_DESCRIPTION}
+        {BRIEF_DESCRIPTION}.
       </Heading>
-      <Flex py="6" justify="center" align="center">
+      <Flex justify="center" align="center">
         <NextLink href="/comparator" passHref>
           <Button as="a" variant="cta">
             Try me now!
           </Button>
         </NextLink>
       </Flex>
     </Stack>
-    <Image src={mascotLogo} alt="" quality={100} />
+    <Center width="full">
+      <Image src={mascotLogo} alt="" quality={100} />
+    </Center>
   </Stack>
 )
 
+interface FeatureItemProps {
+  icon: IconType
+  title: string
+  children: ReactNode
+}
+
+const FEATURES_DESCRIPTIONS: Array<FeatureItemProps> = [
+  {
+    icon: HiOutlineSwitchHorizontal,
+    title: 'Compare releases easily',
+    children: (
+      <Box>
+        Sifting through changelogs on GitHub taking too much time?
+        <br />
+        Let Octoclairvoyant put the list of changes in a single view!
+      </Box>
+    ),
+  },
+  {
+    icon: HiOutlineShare,
+    title: 'Share changelogs',
+    children: (
+      <Box>
+        Want to let your team review the changes in a dependency?
+        <br />
+        Give them a link!
+      </Box>
+    ),
+  },
+  {
+    icon: HiOutlineFire,
+    title: "Don't miss breaking changes",
+    children: (
+      <Box>
+        Octoclairvoyant finds all breaking changes, and lists them at the top.
+        <br />
+        You can’t miss those pesky gotcha’s now!
+      </Box>
+    ),
+  },
+  {
+    icon: HiOutlineFilter,
+    title: 'No manual sorting',
+    children: (
+      <Box>
+        Want a list of major, minor and patch level changes?
+        <br />
+        Octoclairvoyant groups changes into categories for you!
+      </Box>
+    ),
+  },
+  {
+    icon: HiOutlineTag,
+    title: 'Changes per version',
+    children: (
+      <Box>
+        Want to know which version introduced a certain change?
+        <br />
+        Octoclairvoyant labels each change with the version number.
+      </Box>
+    ),
+  },
+]
+
+const FeatureItem = ({ icon, title, children }: FeatureItemProps) => {
+  return (
+    <Box>
+      <Flex
+        boxSize={10}
+        bgColor="secondary.200"
```

Comment:
On Figma, the dark mode:

- uses `secondary.800` for the round background fill
- uses `secondary.200` for the icon itself

Right now the dark mode has the same icons as the light mode, which makes them "vibrate" when I look at them. 😄

Can you try to fix this?

Reply:
Sure! I completely forgot about the dark mode, I'll fix it later.

Reply:
Cool!

Do you want to merge what we have already? You can then work on the dark mode changes in a separate PR?

Or do you want to do all the frontpage work at once in this PR?